ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyOptimizer.java

```diff
@@ -437,7 +437,7 @@ private void transitionLayoutOnlyViewToNativeView(
 
   private static boolean isLayoutOnlyAndCollapsable(@Nullable ReactStylesDiffMap props) {
     if (props == null) {
-      return true;
+      return false;
```

Comment:
Ok this is actually harder than I thought because just adding `collapsable=false` doesn't really work because children are not actually `View` components so adding the `collapsable` prop doesn't do anything. In the case of `ListView` it's always a `StaticRenderer` which ignores the `collapsable` prop. I'm kind of leaning back towards adding the `collapsableChildren` prop to `View` that ensures that all it's children map to a native `View` and don't get optimized out.

Here's a simple example to show why adding  the `collapsable` props to `ScrollView` children in JS doesn't work:
``` js
const CustomView = ({ text }) => <View><Text>{text}</Text></View>;

<ScrollView>
   // Works, view gets rendered with `collapsable=false`
  <View />
  // Doesn't work because the collapsable prop gets passed to `CustomView` but is not 
  // passed down to it's `View` child
  <CustomView text="hello" />
</ScrollView>
```